### DISCUSSION TITLE
Are pomodoros not supposed to show up on the timeline?

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [almarklein/timetagger#372] [Q&A] [answered]
I'm confused about this so bear with me if this is too silly a question. I enabled pomodoro support and completed a pomodoro and the break following it. It was my assumption that the pomodoro would show up in the timeline as a 25 minutes block while the break wouldn't but right now, neither does. Is this expected behaviour or did I miss something? 

I'm running v23.4.1 on Arch Linux. Lemme know if there are any other details I could provide.

**Comment by almarklein** [comment]
In understand the confusion! No, the pomodoro is intended as a tool to help you focus, which is turned on/off automatically, but the pomo blocks (work/break) do not affect the timeline.

**Comment by lokesh-krishna** [comment]
Thanks for clarifying! So I'll need to keep track of the pomo blocks manually, right?

**Reply by almarklein** [reply, marked as answer]
To be honest, I would just count pomo-breaks as work-time, since they are important to stay focused in the 25min blocks.

If you do want to only count the 25 min blocks, then yes, you'd have to do that manually. I could add an option to the list of possible pomo improvements in #115, to let the pomo timer start/stop records. Also see #124 for a somewhat related issue.

**Reply by lokesh-krishna** [reply]
I do think adding blocks using pomo timer start/stop would be nice! Thanks for your patience and for such an amazing tool!
